src/Akka.Persistence.Sql/Query/Dao/BaseByteReadArrayJournalDao.cs

```diff
@@ -146,24 +152,25 @@ orderby r.Ordering
             => Task.FromResult(
                 AsyncSource<JournalRow>
                     .FromEnumerable(
-                        new { _connectionFactory = ConnectionFactory, persistenceId, fromSequenceNr, toSequenceNr, toTake = MaxTake(max) },
+                        new {  persistenceId, fromSequenceNr, toSequenceNr, toTake = MaxTake(max) },
```

Comment:
This whole bit is making me ask if we need a better way to capture certain 'needful' for these captures.
What I'm wondering is, should we have a helper class like:

```
public class CaptureHolder
{
     public readonly AkkaPersistenceDataConnectionFactory;
     public readonly ??? ReadIsolationLevel;
     public readonly CancellationToken ShutdownToken;
     public readonly TagMode Mode;
}
```

Then, we can pass that reference, instead of capturing `this`...
And frankly that might help further 'separate' the query logic out for the sake of testability. :)